packages/extension/src/ui/action/views/swap/views/swap-best-offer/components/swap-best-offer-block/index.vue

```diff
@@ -67,7 +67,7 @@
         Offer includes
         {{ $filters.formatFloatingPointValue(toReadableAdditionalFees).value }}
         {{ network?.currencyName.toUpperCase() }}
-        Bridging fee
+        {{ pickedTrade.additionalNativeFeesDescription || "Bridging fee" }}
```

Comment:
I would keep it as `additional native fees`, bridging is technically different. Bridging is when you transfer tokens cross chains